src/Properties/ModelPropertyExtension.php

```diff
@@ -155,9 +155,15 @@ private function requireFiles(array $files): void
 
     private function getDateClass(): string
     {
-        return $this->dateClass ?? class_exists(\Illuminate\Support\Facades\Date::class)
-            ? '\\'.get_class(\Illuminate\Support\Facades\Date::now())
-            : '\Illuminate\Support\Carbon';
+        if (! $this->dateClass) {
+            $this->dateClass = class_exists(\Illuminate\Support\Facades\Date::class)
+                ? '\\'.get_class(\Illuminate\Support\Facades\Date::now())
+                : '\Illuminate\Support\Carbon';
+
+            $this->dateClass .= '|Carbon\Carbon';
```

Comment:
```suggestion
            $this->dateClass .= '|\Carbon\Carbon';
```

I think it should be `\Carbon\Carbon`

Reply:
Thank you :) totally missed it